lib/libutee/include/remoteproc_pta.h

```diff
@@ -0,0 +1,179 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (C) 2023, STMicroelectronics - All Rights Reserved
+ */
+
+#ifndef __REMOTEPROC_PTA_H
+#define __REMOTEPROC_PTA_H
+
+#include <util.h>
+
+/*
+ * Interface to the pseudo TA which provides platform implementation
+ * of the remote processor management
+ */
+
+#define PTA_RPROC_UUID { 0x54af4a68, 0x19be, 0x40d7, \
+		{ 0xbb, 0xe6, 0x89, 0x50, 0x35, 0x0a, 0x87, 0x44 } }
+
+/* Firmware format */
+#define PTA_RPROC_PROPRIETARY_FMT		BIT32(0)
+#define PTA_RPROC_ELF_FMT			BIT32(1)
+
+/* Firmware image protection */
+/* The platform supports copy of the input firmware image in secure memory */
+#define PTA_RPROC_FW_SECURE_COPY		BIT32(0)
+/* The platform supports load of segment with hash protection */
+#define PTA_RPROC_FW_WITH_HASH_TABLE	BIT32(1)
+/* The platform is able to change access to secure the firmware input image */
+#define PTA_RPROC_FW_MEMORY_PROTECTION	BIT32(2)
+
+/* Platform predefined TLV ID */
+/* boot address of secure fw */
+#define PTA_RPROC_TLV_SBOOTADDR            U(0x21)
+/* boot address of non-secure fw */
+#define PTA_RPROC_TLV_NSBOOTADDR           U(0x22)
+
+/* Platform predefined TLV LENGTH (byte) */
+#define PTA_RPROC_TLV_SBOOTADDR_LGTH       U(4)
+#define PTA_RPROC_TLV_NSBOOTADDR_LGTH      U(4)
```

Comment:
prefer tabs over space chars in the 4 macro definitions above

Reply:
Done